File: Assets/AltUnityTester/AltUnityDriver/Commands/InputActions/ScrollMouse.cs
public class ScrollMouseDriver : AltBaseCommand
{
    float speed;
    float duration;
    public ScrollMouseDriver(SocketSettings socketSettings, float speed, float duration) : base(socketSettings)
    {
        this.speed = speed;
        this.duration = duration;
    }
    public void Execute()
    {
        Socket.Client.Send(toBytes(CreateCommand("scrollMouse", speed.ToString(), duration.ToString())));
        var data = Recvall();
        if (data.Equals("Ok"))
            return;
        HandleErrors(data);
    }
}

File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings~/python/altunityrunner/commands/InputActions/scroll_mouse.py
from altunityrunner.commands.command_returning_alt_elements import CommandReturningAltElements
import time
class ScrollMouse(CommandReturningAltElements):
    def __init__(self, socket,request_separator,request_end, speed, duration):
        super().__init__(socket,request_separator,request_end)
        self.speed=speed
        self.duration=duration
    
    def execute(self):
        print ('Scroll mouse with: ' + str(self.speed))
        data = self.send_data(self.create_command('scrollMouse', self.speed, self.duration ))
        return self.handle_errors(data)